frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -662,8 +662,8 @@ To:
         text = _("File search"),
         help_text = _([[Search a book by filename in the current or home folder and its subfolders.
 
-A search for '*' will show all files.
-Search string supports Lua patterns.
+Wildcards for one '?' or more '*' letters can be used.
```

Comment:
Sounds a bit odd. Not really satisfied with these:
```
Wildcards like '?' or '*' can be used to match one or many characters.
Use wildcards like '?' to match any single character, and '*' to match multple any characters.
```
I'll wait for our english doctors to suggest something better :)